.azure/pipelines/stress.yml

```diff
@@ -0,0 +1,40 @@
+# This configuration builds the repository and runs stress
+pr: none
+
+# Don't run CI for this config
+trigger:
+  batch: true
+  branches:
+    include:
+    - 'jkotalik/stressRuns'
+    - 'master'
```

Comment:
do we want this to run on every merge into 'master'?

Reply:
nope, I just fixed it to be a scheduled build every 3 hours.